.github/workflows/tools_tests.yaml

```diff
@@ -88,4 +88,19 @@ jobs:
       run: |
         $DOCKER_LOGIN
         docker push $DOCKER_REGISTRY/openpilot-docs-base:latest
-        docker push $DOCKER_REGISTRY/openpilot-docs:latest
\ No newline at end of file
+        docker push $DOCKER_REGISTRY/openpilot-docs:latest
+
+  devcontainer:
+    name: devcontainer
+    runs-on: ubuntu-latest
+    if: github.repository == 'commaai/openpilot'
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - name: Setup Dev Container CLI
+      run: npm install -g @devcontainers/cli
+    - name: Build dev container image
+      run: devcontainer build --workspace-folder .
+    - name: Run dev container
+      run: devcontainer up --workspace-folder .
```

Comment:
can we also run something like "scons --dry-run" to get some coverage on the environment?